lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -146,6 +146,19 @@ <h3 class="hd hd-3" id="header-${ section_data['section_key'] }">${ section_data
   </div>
 </div>
 
+<%static:optional_include_mako file="google_analytics.html" is_theming_enabled="True" />
+<% ga_acct = static.get_value("GOOGLE_ANALYTICS_ACCOUNT", settings.GOOGLE_ANALYTICS_ACCOUNT) %>
+% if ga_acct:
+  <script type="text/javascript>
+    jQuery(document).ready(function () {
+      jQuery('.menu a').click(function(){
```

Comment:
Did you manage to test this?
Superficially, I don't suspect this will work.

It looks like we did a little too much copy/paste w/out adapting the snippet to work on our site;
it was just provided as an example of the approach to use, not meant to be the final implementation.

Specifically, that CSS selector (`.menu a`) doesn't appear to match any elements on the instructor dash..